plugins/reach/lib/kReachManager.php

```diff
@@ -95,14 +95,14 @@ private function addingEntryVendorTaskByObjectIds($entryId, $allowedCatalogItemI
 
 	private function shouldSkipAutoRule($object, $entryId, $featureType)
 	{
-		if ($featureType == VendorServiceFeature::CAPTIONS || $featureType == VendorServiceFeature::TRANSLATION)
+		if ($featureType == VendorServiceFeature::CAPTIONS)
 		{
 			if ($object instanceof entry && $object->getBlockAutoTranscript())
 			{
 				KalturaLog::log("Skip the entry automatic rule if it's a caption or transcript and 'Block Auto Transcript' is enabled");
 				return true;
 			}
-			if ($object instanceof categoryEntry)
+			else
```

Comment:
fix the log message 

you can use a code like :
 
$entry = $object instanceof entry ? $object : null; 
if (!$entry) 
{
$entry = entryPeer::retrieveByPK($entryId);
}

if ($entry && $entry->getBlockAutoTranscript())
{
KalturaLog::log("Skip automatic rule for captions for $entryId 'Block Auto Transcript' is enabled") // something like that
return true;
}